ideascube/mediacenter/management/commands/clean.py

```diff
@@ -17,13 +17,19 @@ def add_arguments(self, parser):
         subs = parser.add_subparsers(
             title='Commands', dest='cmd', metavar='',
             parser_class=argparse.ArgumentParser)
+        parser.add_argument('--dry-run', action='store_true',
+                            help='Print the list of medias that would be '
+                                 'removed. Do not actually remove them')
+
         clean_leftover = subs.add_parser(
             'leftover-files',
             help='Clean mediacenter files not associated with a document.')
         clean_leftover.set_defaults(func=self.clean_leftover)
-        parser.add_argument('--dry-run', action='store_true',
-                            help='Print the list of medias that would be '
-                                 'removed. Do not actually remove them')
```

Comment:
This means the command now is:

```
$ ideascube clean --dry-run leftover-files
$ ideascube clean --dry-run media
```

However, the `--dry-run` option applies to the subcommand, so I'd much prefer we had:

```
$ ideascube clean leftover-files --dry-run
$ ideascube clean clean --dry-run
```

You can do that with [a common parser](https://github.com/ideascube/ideascube/blob/87aa1c444dde784fd7fbf6ae3a09d040e2046bd8/ideascube/serveradmin/management/commands/catalog.py#L28-L33) and have it be [a parent of the commands](https://github.com/ideascube/ideascube/blob/87aa1c444dde784fd7fbf6ae3a09d040e2046bd8/ideascube/serveradmin/management/commands/catalog.py#L59).

It's a better UX, because it means you can run:

```
$ ideascube catalog clean media --dry-run
```

And then quickly press ↑ followed by `ctrl`+`w` then ↵.

(it would also be an API break if we had released the existing command, or if the Ansiblecube folks had already created the cron job for the `leftover-files`)